cf/read_write/um/umread.py

```diff
@@ -662,6 +662,14 @@ def __init__(
         BDX = real_hdr[bdx]
         BDY = real_hdr[bdy]
 
+        if not LBROW or not LBNPT:
+            logger.warn(
+                f"WARNING: Skipping field derived from STASH code {stash} "
+                f"with LBROW={LBROW} and LBNPT={LBNPT}"
```

Comment:
Could we add a brief sentence to justify _why_ the field(s) was (were) skipped to put the line into context? Or should that be obvious to users of UM and PP data (those who know more about the lookup codes than myself - all I can find from a little look-see is some descriptions of those pair of entries in the UM docs e.g. [here](https://artefacts.ceda.ac.uk/badc_datadocs/um/umdp_F3-UMDPF3.pdf) to add to your comment on this Issue:

> Such fields are runlength encoded, and cf-python can't unpack them because it doesn't know what the mask is.

as background)?

Reply:
Fair enough. That all makes sense and thanks for clarifying.

In that case, it sounds like indicating it is potentially run-length encoded is indeed a good way forward, so please go ahead with that suggestion or a similar one.

Reply:
(Discussed offline. In short, all good and new commit implements, so marking as resolved.)